docs/guides/web-ui/explore.md

```diff
@@ -26,40 +26,7 @@ WHERE is_exception
 You can run more complex queries as well, using subqueries, CTEs, joins, aggregations, custom expressions,
 and any other standard SQL.
 
-## Records Schema
-
-The schema of the `records` table is:
-
-```sql
-CREATE TABLE records AS (
-    start_timestamp timestamp with time zone,
-    created_at timestamp with time zone,
-    trace_id text,
-    span_id text,
-    parent_span_id text,
-    kind span_kind,
-    end_timestamp timestamp with time zone,
-    level smallint,
-    span_name text,
-    message text,
-    attributes_json_schema text,
-    attributes jsonb,
-    tags text[],
-    otel_links jsonb,
-    otel_events jsonb,
-    is_exception boolean,
-    otel_status_code status_code,
-    otel_status_message text,
-    otel_scope_name text,
-    otel_scope_version text,
-    otel_scope_attributes jsonb,
-    service_namespace text,
-    service_name text,
-    service_version text,
-    service_instance_id text,
-    process_pid integer
-)
-```
```

Comment:
Can we put this into into the new page? I'm linking it in my dashboards PR so people can glance quickly which columns are available

Reply:
I don't like this form, it's overwhelming and unhelpful, and the types don't really help. The sidebar in the reference page names all the important columns, categorised and roughly ordered by importance, with links to each description.

<img width="286" alt="Screenshot 2025-06-24 at 20 26 51" src="https://github.com/user-attachments/assets/4663d613-3e54-4a66-8c76-95d17c47aa6c" />